internal/service/rds/cluster_activity_stream_test.go

```diff
@@ -0,0 +1,227 @@
+package rds_test
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/rds"
+	sdkacctest "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	tfrds "github.com/hashicorp/terraform-provider-aws/internal/service/rds"
+	"github.com/hashicorp/terraform-provider-aws/internal/tfresource"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_rds_cluster_activity_stream", &resource.Sweeper{
+		Name: "aws_rds_cluster_activity_stream",
+		F:    func(region string) error { return nil },
+		Dependencies: []string{
```

Comment:
sweeper should sit in dedicated sweeper.go file in the rds package
also the dep is the other way around. (if needed in the first place) you want the parent resources to be depend on the child resources to be sweeped

Reply:
Moved and removed the dependencies.